routes/message.js

```diff
@@ -0,0 +1,16 @@
+const router = require('express').Router()
+
+const asyncWrapper = require('~/middlewares/asyncWrapper')
+const { authMiddleware } = require('~/middlewares/auth')
+const isEntityValid = require('~/middlewares/entityValidation')
+
+const messageController = require('~/controllers/message')
+const Message = require('~/models/message')
+
+const body = [{ model: Message, idName: 'message' }]
```

Comment:
here must be a model which you pass in body. I don`t think that its message

Reply:
Fixed.